pkg/reconciler/trigger/resources/dispatcher.go

```diff
@@ -198,6 +198,13 @@ func MakeDispatcherDeployment(args *DispatcherArgs) *appsv1.Deployment {
 				Value: *args.Subscriber.CACerts,
 			})
 	}
+	if args.DLX {
+		dispatcher.Env = append(dispatcher.Env,
+			corev1.EnvVar{
+				Name:  "DLX",
+				Value: "true",
```

Comment:
Since this is the Trigger's Dispatcher his should have the value of args.DLX

This one you can put it on the pkg/reconciler/brokker/resources/dispatcher.go that is always going to be true

Reply:
Env variable will only be set if DLX is true. Defaults to false